README.md

```diff
@@ -1,6 +1,6 @@
 # Docker Extensions
 
-This repository includes all resources to create Docker Extensions.
+This repository includes extension CLI and samples to create Docker Extensions.
```

Comment:
Similar to when we use the term "Docker Extensions", I think we should use the term "Extensions CLI". Brand names are typically capitalized. 

@aevesdocker could you confirm? 

```suggestion
This repository includes the Extensions CLI and samples to create Docker Extensions.
```

Reply:
Yes agreed @felipecruz91, thanks for catching that